style.css

```diff
@@ -97,3 +112,178 @@ li:hover {
 li:active {
   color: #2230d2;
 }
+
+/* work section */
+.work {
+  background-color: #e5e5e5;
+  margin: 100px 0;
+  align-items: center;
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.container {
```

Comment:
- [ ] great job with the project .please make sure the different content of projects are displayed with the flexbox

Reply:
changed it to flex with column